core/src.ts/web/examples/get_key_info.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <!--
+    LibHaLo - Programmatically interact with HaLo tags from the web browser, mobile application or the desktop.
+    Copyright by Arx Research, Inc., a Delaware corporation
+    License: MIT
+    -->
+        <title>LibHaLo Demo</title>
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+          integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+
+        <script type="text/javascript">
+            // ensure the library is always fully reloaded
+            document.write('<script src="../dist/libhalo.js?_v=' + (
+            Math.random() + '') + '"></scr' + 'ipt>');
+        </script>
+    </head>
+    <body>
+        <div class="container">
+            <h1>LibHaLo Demo</h1>
+            <p class="text-muted">
+                Get information about the specific key slot.
+            </p>
+            <div class="mb-3">
+                <label class="form-label">Override the default URL in QR code</label>
+                <input type="text" class="form-control" id="keyNo" value="1" placeHolder="e.g. 2">
+                <p class="text-muted">Only for debugging. It's recommended to keep this as it is.</p>
```

Comment:
Wrong text labels.

Reply:
Right, tweaked it here: https://github.com/arx-research/libhalo/pull/403/commits/04b27f854901f212ae45c88ec40250b60275561c